pipelines/components/tensorflow/kmeansclustering/local_run.sh

```diff
@@ -0,0 +1,10 @@
+docker run -v $GOOGLE_APPLICATION_CREDENTIALS:/credentials.json -e "GOOGLE_APPLICATION_CREDENTIALS=/credentials.json" <TAG> \
```

Comment:
Are we adding Docker stuff to our code? I thought our deliverable is just the python project.